packages/components/Checkbox/src/Checkbox.types.ts

```diff
@@ -183,7 +188,7 @@ export interface CheckboxInfo {
 
 export interface CheckboxSlotProps {
   root: React.PropsWithRef<PressablePropsExtended>;
-  checkbox: IViewProps;
+  checkbox: PressableProps;
```

Comment:
Might be good to copy what I did the line above and use `PressablePropsExtended` so that we also support desktop specific Pressable props.

Reply:
Changed.